crates/re_viewer/src/ui/time_panel/time_axis.rs

```diff
@@ -113,18 +99,58 @@ fn gap_size_heuristic<T>(time_type: TimeType, times: &BTreeMap<TimeInt, T>) -> u
     gap_threshold
 }
 
-/// Collapse any gaps larger or equals to the given threshold.
-fn create_ranges<T>(times: &BTreeMap<TimeInt, T>, gap_threshold: u64) -> vec1::Vec1<TimeRange> {
+fn collect_gaps(times: &TimeHistogram, min_gap_size: u64, max_collapses: usize) -> Vec<u64> {
```

Comment:
```suggestion
fn collect_candidate_gaps(times: &TimeHistogram, min_gap_size: u64, max_collapses: usize) -> Vec<u64> {
```
Maybe rename to make this more clear these gaps won't necessarily be used for splits.